docs/contributing/commit-signing.mdx

```diff
@@ -16,61 +16,144 @@ For example, on 3 August 2022, Stephen Lacy
 digitally signed).
 
-To protect against commit spoofing, all Bitwarden contributors are encouraged to digitally sign
-their commits.
+To protect against commit spoofing, <strong>all Bitwarden contributors are encouraged to digitally
+sign their commits</strong>.
 
 ## Setting up commit signing
 
-Github supports commit signing with GPG, SSH and S/MIME.
+GitHub supports [commit signing][github-verification] with GPG, SSH and S/MIME.
 
 :::tip
 
-If you're unsure what to use, we recommend you create a commit signing key using SSH. Remember to
-protect the key with a [strong passphrase or password][password-generation]. Alternatively set up
-your [yubikey as your signing key](#hardware-backed-ssh-keys).
-
-```bash
-# path to save your private signing key
-KEY_FILE=~/.ssh/bw-signing
-ssh-keygen -f $KEY_FILE -C "$(git config --global --get user.email)" -t ed25519
-```
+If you're unsure what to use, <strong>we recommend you create a commit signing key using SSH per
+latest security best practices </strong> (see the
+[PGP problem](https://www.latacora.com/blog/2019/07/16/the-pgp-problem/) for more details).
 
 :::
 
-### Configuration
+### Standard SSH Key Configuration
 
-1. Configure git to sign using SSH.
+1. Ensure you have globally configured your git email address:
+
+   ```bash
+   git config --global user.email "you@example.com"
+   ```
+
+2. Generate an SSH key for commit signing:
+
+   ```bash
+   # path to save your private signing key
+   KEY_FILE=~/.ssh/bw-signing
+   ssh-keygen -f $KEY_FILE -C "$(git config --global --get user.email)" -t ed25519
+   ```
+
+   :::tip
+
+   Remember to protect the key with a [strong passphrase or password][password-generation].
+   Alternatively set up your [yubikey as your signing key](#hardware-backed-ssh-key-configuration).
+
+   :::
+
+3. Configure git to sign using SSH.
 
    ```bash
    git config --global gpg.format ssh
    git config --global user.signingkey "${KEY_FILE}.pub"
    ```
 
-2. Follow the [Github documentation][github-verification] to configure commit signing
+4. Follow the
+   [SSH GitHub documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#ssh-commit-signature-verification)
+   to configure commit signing
 
-3. Configure your preferred git tool below
+5. Configure your preferred git tool below ([CLI](#commit-signing-with-git-cli),
+   [VS Code](#commit-signing-with-visual-studio-code),
+   [SourceTree](#commit-signing-with-sourcetree))
 
-4. Push a test commit to Github and ensure that the "Verified" badge appears next to the commit
+6. Push a test commit to GitHub and ensure that the "Verified" badge appears next to the commit
    description:
 
-   ![Image showing the Verified badge in Github](./commit-signing.png)
+   ![Image showing the Verified badge in GitHub](./commit-signing.png)
 
-### Command Line
+7. (Optional) See Avoiding SSH Key Passphrase Prompts Per Commit below to avoid being prompted for
```

Comment:
Can this be linked to the section header?

Reply:
f2d0bb1724a62d577c0104815ec4c45f2f074029